input/pagecontent/2-4-Subscribing.md

```diff
@@ -120,7 +119,7 @@ Although these event notifications are triggered by a successful subscription, t
   
 ### Subscription Denial
 
-If (and when) a subscription is denied, the Hub SHALL inform the Subscriber. This can occur when a subscription is requested for a variety of reasons, or it can occur after a subscription had already been accepted because the Hub no longer supports that subscription (e.g. it has expired).
+If (and when) a subscription is denied, the Hub SHALL inform the Subscriber. This can occur for a variety of reasons. Both when a subscription is requested, or after a subscription has already been accepted because the Hub no longer supports that subscription (e.g. it has expired).

```

Comment:
```suggestion
If (and when) a subscription is denied, the Hub SHALL inform the Subscriber. A subscription can be denied when it is  requested or even after having been accepted (e.g. the subscription has expired).

```
The extant sentence is explicitly about chronology, it's best to keep it about that. Simply language.